src/targets/services/cleanOrphanAccounts.js

```diff
@@ -0,0 +1,81 @@
+const cozyFetch = require('../../lib/services/cozyFetch')
+
+const logMessagePrefix = level =>
+  `[cozy-collect][cleanOrphanAccounts][${level}]:`
+
+const output = level =>
+  function() {
+    console.log.apply(
+      console,
+      [logMessagePrefix(level)].concat(
+        // Node < 7 does not know about Object.values
+        Object.keys(arguments).map(key => arguments[key])
```

Comment:
Services and konnectors are run one Node 8.

Reply:
Mind -> blown.

So what is run on node 6 now ? Nothing ? I guess we can all upgrade or node local version to node 8 without side effects on existing applications ?

Reply:
Yes nothing is run on Node 6 and konnectors/services should be run with node 8 on local machines.

Reply:
Great news !